CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Minor
 
+- Icon: Add new replace/scale icons (#567)
```

Comment:
@loserford Could we move this to the bottom of this list? We always try to keep these in order of the PR #

Reply:
updated.